opcodes/riscv-opc.c

```diff
@@ -686,3 +686,29 @@ const struct riscv_opcode riscv_opcodes[] =
 /* Terminate the list.  */
 {0, 0, 0, 0, 0, 0, 0}
 };
+
+const struct riscv_opcode riscv_insn_types[] =
+{
+/* name,  isa,           operands, match, mask,    match_func, pinfo.  */
+{"r",     "I",   "O4,F3,F7,d,s,t",     0,    0,  match_opcode, 0 },
+{"i",     "I",   "O4,F3,F7,d,s,j",     0,    0,  match_opcode, 0 },
+{"s",     "I",   "O4,F3,F7,t,s,q",     0,    0,  match_opcode, 0 },
+{"s",     "I",  "O4,F3,F7,t,q(s)",     0,    0,  match_opcode, 0 },
+{"sb",    "I",  "O4,F3,F7,d,o(s)",     0,    0,  match_opcode, 0 },
+{"u",     "I",     "O4,F3,F7,d,u",     0,    0,  match_opcode, 0 },
+{"uj",    "I",     "O4,F3,F7,d,a",     0,    0,  match_opcode, 0 },
```

Comment:
```sb``` and ```uj``` should be renamed ```b``` and ```j``` (the names changed in v2.2 of the spec).

Reply:
Updated